app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt

```diff
@@ -78,4 +86,34 @@ abstract class BaseFragment<B : ViewBinding, V : BaseView> : Fragment(), BaseVie
     internal abstract fun createBinding(): B
 }
 
-interface KeyboardUsingFragment
\ No newline at end of file
+@Suppress("DEPRECATION")
```

Comment:
Why do we use this deprecation?

Reply:
Below api 30 there is only the deprecated way for setting light systemBar, as far as I can see